tests/skr/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  proxy:
+    image: $REGISTRY/skr/proxy:$TAG
+    network_mode: "host"
+    build:
+      context: .
+      dockerfile: proxy.Dockerfile
+  http_sidecar:
+    image: $REGISTRY/skr/http_sidecar:$TAG
```

Comment:
You're right, the only reason to split them here would be to make running them locally easier but since they rely on being in ACI I guess that's not a likely use case

Reply:
https://github.com/microsoft/confidential-sidecar-containers/pull/117/commits/176edbded9fee3240539a4b70607c9c8278cd6aa